config/locales/en.yml

```diff
@@ -1292,6 +1292,9 @@ en:
     create:
       success_flash: "Successfully added sample group <strong>%{sample_group}</strong> to team <strong>%{team}</strong>."
   system_notifications:
+    emails:
+      title: "What's new notification"
+      intro_paragraph: "Hello %{user_name}, you received new What's new notification!"
```

Comment:
Maybe change intro paragraph, cause it sound strange 'new What's new'...
Maybe like this - "you received new system notification!"

Reply:
True, thanks! I will send a screenshot of mail template to Matjaz and Miha, they probably find a right copy :)

Reply:
Agreed. I attached screenshots of emails to JIRA ticket. Matjaz and Darja will update copy I guess.